ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent.json

```diff
@@ -1055,6 +1055,16 @@
       {
         "UserRoles": [
           "[LASOLICITOR]",
+          "[SOLICITOR]",
+          "[solicitorA]",
+          "[solicitorB]",
+          "[solicitorC]",
+          "[solicitorD]",
+          "[solicitorE]",
+          "[solicitorF]",
+          "[solicitorG]",
+          "[solicitorH]",
+          "[solicitorI]",
```

Comment:
@aliveni we suspect these may need to be captilised. Let me confirm again once tested on demo.

Reply:
@Jamesnelson117  updated the case roles and added SOLICITORJ